contracts/gateway-v1.yaml

```diff
@@ -2136,6 +2136,125 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /hip-links/link:
+    post:
+      tags:
+        - link
+        - hip facing
+      summary: Link patient's care contexts
+      description: >
+        Request from Hip to link a patient's care contexts
+          1. **Validate account reference number and care context reference number**
+          2. **Before eventual link confirmation, CM needs to authenticate the request with the patient**
+      parameters:
```

Comment:
Don't we need a header X-CM-ID (X-CM-ID=ncg)?

Reply:
I think some time in between ganeshan said we have to move these attributes inside request body. So, I moved it inside. If you want I can move it outside